### PR TITLE
Apb 6119 add search by group name

### PR DESCRIPTION
**Pull request by geoffreywatson**

### CODE REVIEW

File: app/views/groups/manage/dashboard.scala.html [hunk 1]
```diff
@@ -24,22 +24,59 @@
...
+        </div>
+        <div class="govuk-grid-column-one-third" id="filter-buttons-wrapper">
+            @submit_button_group()
+        </div>
+        <div class="govuk-grid-column-one-third">
+            @input_select(
+                field = filterGroupSummaryForm("sort"),
+                label = msgs("Sort by"),
+                options = Seq(("default" ,"Default (A-Z)"), ("reverse", "Z-A")),
+                firstItem =  Seq.empty[SelectItem]
```
Comment (CarolineS-QA) [inline app/views/groups/manage/dashboard.scala.html:70]: you shouldn't need this line if you've made it the default in the component

**Review by CarolineS-QA — COMMENTED** [review]
Also, are we leaving the filter (dropdown) non-functional for now? PR Looks okay overall.

File: app/views/groups/manage/dashboard.scala.html (hunk 1 above)
Comment (geoffreywatson) [reply app/views/groups/manage/dashboard.scala.html:70]: which line please? then I'll remove, thx

Comment (CarolineS-QA) [reply app/views/groups/manage/dashboard.scala.html:70]: line 70, sorry I thought I selected it

Comment (geoffreywatson) [reply app/views/groups/manage/dashboard.scala.html]: ok removed now please check

**Pull request merged**